doc/migration/version-upgrade-guide.md

```diff
@@ -486,6 +486,75 @@ All method signatures for [`sigs.k8s.io/controller-runtime/pkg/client.Client`](h
     - Delete the old CRD manifest files.
     - Rename CR manifest file names from `<group>_<version>_<kind>_cr.yaml` to `<full group>_<version>_<kind>_cr.yaml`.
 
+## `v0.12.x`
+
+### Go version
+
+- Ensure that you are using a go version 1.13+
+
+### modules
+
+- Replace your `go.mod` file for:
```

Comment:
It doesn't look like we have been telling users to do a full replacement of their dependency files, and for good reason, I think. Users may have made their own custom changes to bring in more dependencies. This instruction could cause their custom dependency inclusions to break or change in unexpected ways.

Rather, we should document which pieces of our `go.mod` scaffold changed between `v0.11.x` and `v0.12.x`.